tasks/notify.py

```diff
@@ -97,6 +97,8 @@ def send_message(ctx, notification_type="merge", print_to_stdout=False):
     base = base_message(header, state)
 
     # Send messages
+    metrics = []
+    timestamp = int(datetime.now().timestamp())
```

Comment:
```suggestion
    timestamp = int(datetime.now(timezone.utc).timestamp())
```
I'm not 100% sure if we should always sent datetime in utc timezone